tests/browserstack/specs/ballotMainTest.js

```diff
@@ -108,85 +108,134 @@ describe('Basic cross-platform We Vote test',  () => {
 
     // //////////////////////
     // Change Address
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('changeAddressHeaderBar'); // Open the "Change Address" modal
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('profileCloseSelectBallotModal');
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-
-    // //////////////////////
-    // We want to start by setting the location, which will automatically choose the next upcoming election for that address
-    await simpleClick('changeAddressHeaderBar'); // Opens the "Enter Your Full Address" link
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-
-    if (isIOS && isCordovaFromAppStore) {
-      await setNewAddressIOS('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
-    } else if (isAndroid && isCordovaFromAppStore) {
-      await setNewAddressAndroid('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
-    } else {
+    if (isDesktopScreenSize) {  // Desktop/tablet version
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      await simpleClick('changeAddressOrElectionHeaderBar'); // Open the "Change Address" modal
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
       await setNewAddress('addressBoxText', 'Redmond, WA 98052'); // Sets the text for the address box and hits enter
+      await simpleClick('changeAddressOrElectionHeaderBar'); // Open the "Change Address" modal
```

Comment:
I think this will want to be moved down to below the comment "Switch to a known election": `await simpleClick('changeAddressOrElectionHeaderBar'); // Open the "Change Address" modal`